blob/blob.go

```diff
@@ -563,22 +563,22 @@ func (b *Bucket) Exists(ctx context.Context, key string) (bool, error) {
 //
 // If the blob does not exist, Attributes returns an error for which
 // gcerrors.Code will return gcerrors.NotFound.
```

Comment:
Should we perhaps remark that the caller should not modify the returned `Attributes` struct?

Reply:
I don't think it matters if they do, as we create a new `blob.Attributes` to return to them, so they have the only reference. It's worth adding to the driver methods though; done.